runtime/gc_modron_startup/mminit.cpp

```diff
@@ -3137,6 +3138,8 @@ gcInitializeDefaults(J9JavaVM* vm)
 
 		/* Try to initialize basic heap structures with the memory parameters we currently have */
 		if (JNI_OK == j9gc_initialize_heap(vm, memoryParameterTable, extensions->memoryMax)) {
+			/* set vm->isVirtualLargeObjectHeapEnabled after extensions->isVirtualLargeObjectHeapEnabled is set in j9gc_initialize_heap(). */
+			vm->isVirtualLargeObjectHeapEnabled = (extensions->isVirtualLargeObjectHeapEnabled) ? TRUE : FALSE;
```

Comment:
can be done even earlier, before while loop
does not really have to be dependent on if heap is successfully created
it's just a config parameter, could be done even right when you set it in GCExt (in Confiugation). it's probably the best that they match all the time during startup